TWLight/applications/models.py

```diff
@@ -102,6 +103,13 @@ class Meta:
     agreement_with_terms_of_use = models.BooleanField(default=False)
     account_email = models.EmailField(blank=True, null=True)
 
+    PROXY_ACCOUNT_LENGTH_CHOICES = ((1, '1 month'), (3, '3 months'), (6, '6 months'), (12, '12 months'))
```

Comment:
Since we already said "number of months" in the form label, this can probably just be a list of numbers. Will save us translating each of these strings.